.env.example

```diff
@@ -270,3 +270,7 @@ VERKKOKAUPPA_NAMESPACE=tilanvaraus
 # Is reservation unit product mapping enabled (uses Celery and Verkkokauppa API)
 # Defaults to false
 #UPDATE_PRODUCT_MAPPING=
+
+# Is reservation unit accounting sync to Verkkokauppa enabled (uses Celery and Verkkokauppa API)
+# Defaults to false
+UPDATE_ACCOUNTING=
```

Comment:
This should have a value of somekind - or to be in commets.

Reply:
Oops! I commented it 👍